lib/appium.js

```diff
@@ -47,6 +48,8 @@ class AppiumDriver extends BaseDriver {
       if (caps.automationName.toLowerCase() === 'selendroid') {
         // but if we do and it is 'Selendroid', act on it
         return SelendroidDriver;
+      } else if ((caps.automationName || '').toLowerCase()  === 'uiautomator2') {
```

Comment:
The `(caps.automationName || '')` is unnecessary, since this is all surrounded by the check for something in `caps.automationName`.